src/UseCases/HandlePayPalPaymentNotification/HandlePayPalPaymentNotificationUseCase.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\UseCases\HandlePayPalPaymentNotification;
+
+use WMDE\Fundraising\Frontend\Infrastructure\PayPalPaymentNotificationHandler;
+
+/**
+ * @license GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class HandlePayPalPaymentNotificationUseCase {
+
+	private $handler;
+
+	public function __construct( PayPalPaymentNotificationHandler $handler ) {
```

Comment:
Will this UC have more than just this handler? I'm wondering what the difference is